packages/mapviewer/src/modules/map/components/cesium/CesiumInteractions.vue

```diff
@@ -233,6 +246,72 @@ function onClick(event) {
     viewer.scene.requestRender()
 }
 
+/**
+ * This function creates a LayerFeature from a KML feature in 3D.
+ *
+ * @param {Object} viewer Cesium viewer
+ * @param {Object} kmlFeature KML feature
+ * @param {Object} kmlLayer KML layer
```

Comment:
You sure that those can't be a bit better "typed"? Or at least point at to the Cesium online documentation to better know what we are dealing with, with them (no need to describe them the same way they are named 😉 )